rust/lance-core/src/io/writer/statistics.rs

```diff
@@ -254,8 +783,69 @@ impl StatisticsBuilder {
         }
     }
 
+    fn string_statistics_appender(&mut self, row: StatisticsRow) {
+        let min_value_builder = self
+            .min_value
+            .as_any_mut()
+            .downcast_mut::<StringBuilder>()
+            .unwrap();
+        let max_value_builder = self
+            .max_value
+            .as_any_mut()
+            .downcast_mut::<StringBuilder>()
+            .unwrap();
+
+        if let ScalarValue::Int64(Some(null_count)) = row.null_count {
+            self.null_count.append_value(null_count);
+        } else {
+            todo!()
```

Comment:
This should go away from once you switch `null_count` back to being an i64.

Reply:
Removed.